mne/export/_edf.py

```diff
@@ -34,14 +59,18 @@ def _export_raw(fname, raw, physical_range, fmt):
     sfreq = int(raw.info['sfreq'])
     n_secs = n_times / sfreq
 
+    # get any filter information applied to the data
+    lowpass = raw.info['lowpass']
+    highpass = raw.info['highpass']
+    linefreq = raw.info['line_freq']
+    filter_str_info = f"HP:{highpass}Hz LP:{lowpass}Hz N:{linefreq}Hz"
+
     # get data in uV
+    # XXX: what about non-EEG data?... How to handle?
     units = dict()
-    if 'eeg' in raw:
-        units['eeg'] = 'uV'
-    if 'ecog' in raw:
-        units['ecog'] = 'uV'
-    if 'seeg' in raw:
-        units['seeg'] = 'uV'
+    units['eeg'] = 'uV'
+    units['ecog'] = 'uV'
+    units['seeg'] = 'uV'
```

Comment:
This can be one line instead of 4
```
units = dict(eeg='uV', ecog='uV', seeg='uV')
```
or 
```
units = {key: 'uV' for key in ('eeg', 'ecog', 'seeg')}
```